aula/apps/sortides/models.py

```diff
@@ -154,13 +154,13 @@ class Sortida(models.Model):
 
     mitja_de_transport = models.CharField(max_length=2, choices=TIPUS_TRANSPORT_CHOICES,help_text=u"Tria el mitjà de transport")
 
-    empresa_de_transport = models.CharField(max_length=250,help_text=u"Indica el nom de l'empresa de transports i número de contracte/pressupost.")
+    empresa_de_transport = models.CharField(max_length=250, null=True, help_text=u"Indica el nom de l'empresa de transports i número de contracte/pressupost.")
 
-    pagament_a_empresa_de_transport = models.CharField(max_length=100,help_text=u"Indica la quantitat que ha de pagar l'institut pel lloguer del bus, o compra de bitllets. Si no ha de pagar res indica-ho, escriu 'res'.")
+    pagament_a_empresa_de_transport = models.CharField(max_length=100, null=True, help_text=u"Indica la quantitat que ha de pagar l'institut pel lloguer del bus, o compra de bitllets. Si no ha de pagar res indica-ho, escriu 'res'.")
 
-    pagament_a_altres_empreses = models.TextField(help_text=u"Indica la quantitat, l'empresa que ha de rebre els diners, el sistema de pagament, el número de contracte i el termini. Si no s'ha de pagar res indica-ho, escriu 'res'.")
+    pagament_a_altres_empreses = models.TextField( null=True, help_text=u"Indica la quantitat, l'empresa que ha de rebre els diners, el sistema de pagament, el número de contracte i el termini. Si no s'ha de pagar res indica-ho, escriu 'res'.")
 
-    feina_per_als_alumnes_aula = models.TextField(help_text=u"Descriu o comenta on els professors trobaran la feina que han de fer els alumnes que es quedin a l'aula. Si no queden alumnes a l'aula indica-ho.")
+    feina_per_als_alumnes_aula = models.TextField( null=True, help_text=u"Descriu o comenta on els professors trobaran la feina que han de fer els alumnes que es quedin a l'aula. Si no queden alumnes a l'aula indica-ho.")
```

Comment:
Podries considerar `blank=True, default=""` en comptes de `null=True` per tal de seguir les [recomanacions de django](https://docs.djangoproject.com/en/5.0/ref/models/fields/#null) ?

>Avoid using [null](https://docs.djangoproject.com/en/5.0/ref/models/fields/#django.db.models.Field.null) on string-based fields such as [CharField](https://docs.djangoproject.com/en/5.0/ref/models/fields/#django.db.models.CharField) and [TextField](https://docs.djangoproject.com/en/5.0/ref/models/fields/#django.db.models.TextField). If a string-based field has null=True, that means it has two possible values for “no data”: NULL, and the empty string. In most cases, it’s redundant to have two possible values for “no data;” the Django convention is to use the empty string, not NULL. One exception is when a [CharField](https://docs.djangoproject.com/en/5.0/ref/models/fields/#django.db.models.CharField) has both unique=True and blank=True set. In this situation, null=True is required to avoid unique constraint violations when saving multiple objects with blank values.

Reply:
Ok. Fet.